uk.co.compucorp.civicrm.contactactionsmenu/info.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<extension key="uk.co.compucorp.civicrm.contactactionsmenu" type="module">
+  <file>contactactionsmenu</file>
+  <name>Contact Actions Menu</name>
+  <description>This extension provides functionality that allows other extensions
+    add menu items to the contact actions menu on contact summary page.</description>
+  <license>AGPL-3.0</license>
+  <maintainer>
+    <author>Compucorp Ltd</author>
+    <email>info@compucorp.co.uk</email>
+  </maintainer>
+  <releaseDate>2017-12-20</releaseDate>
+  <version>1.7.2</version>
+  <develStage>alpha</develStage>
+  <compatibility>
+    <ver>4.7</ver>
+  </compatibility>
+  <civix>
+    <namespace>CRM/Contactactionsmenu</namespace>
```

Comment:
please change this to `CRM/HRContactActionsMenu`

Reply:
I wanted to change it to the camel case style but the CLASS_PREFIX constant in contactactionsmenu.civix.php is different and the class is autogenerated.

Reply:
yes, it's fine to change it. This file is never auto updated after it has been created